### PR TITLE
Add tutorial for symbol and string collection using `/api/collect` endpoint

### PR DESCRIPTION
**Pull request by keshav-space**
- Refrence https://github.com/nexB/purldb/issues/330

### CODE REVIEW

File: docs/source/how-to-guides/tutorial_symbol_and_string_collection.rst [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+        "license_detections": [],
+        "license_clues": [],
+        "percentage_of_license_text": null,
+        "copyrights": [],
+        "holders": [],
+        "authors": [],
+        "package_data": [],
+        "emails": [],
+        "urls": [],
+        "extra_data": {
```
Comment (JonoYang) [inline docs/source/how-to-guides/tutorial_symbol_and_string_collection.rst:189]: I think we should use an example that returns values in the `extra_data` field to give the reader an idea of what kind of values to expect.

File: docs/source/how-to-guides/tutorial_symbol_and_string_collection.rst [hunk 2]
```diff
@@ -0,0 +1,395 @@
...
```
Comment (JonoYang) [inline docs/source/how-to-guides/tutorial_symbol_and_string_collection.rst:338]: See comment on line 189

**Review by JonoYang — CHANGES_REQUESTED** [review]
@keshav-space Instructions are clear, I left some comments about some of the examples not having data in the `extra_data` field.

File: docs/source/how-to-guides/tutorial_symbol_and_string_collection.rst (hunk 1 above)
Comment (keshav-space) [reply docs/source/how-to-guides/tutorial_symbol_and_string_collection.rst]: @JonoYang Fixed, now we're using a longer file for the tutorial, and we have return values for all pipelines.

**Review by JonoYang — CHANGES_REQUESTED** [review]
@keshav-space Thanks for updating the example to include more results, but I think some of the results can be truncated so the doc page isnt so large with example data. Other than that, I think it looks good.

**Comment by keshav-space** [comment]
@JonoYang 

https://github.com/nexB/purldb/assets/44315208/d2dd7bd6-0bac-4ea3-a384-28f336923f81

**Comment by JonoYang** [comment]
@keshav-space The toggle-able file contents looks great, thanks!

**Pull request merged**